apps/vast/README.md

```diff
@@ -22,32 +22,45 @@ python3 -m pip install -r requirements.txt
 
 ## Quick Start
 
-The bridge gets configured via command line arguments and flags. Inspect the
-command help as follows.
+The bridge is configured either via a config file in YAML format or via command
+line arguments and flags. See `config.yaml.example` for an example configuration
+file that make use of [fever alertify](https://github.com/DCSO/fever) to
+transform sighting contexts before they get printed to `STDOUT`. See the section
+[Bridge Features](/tenzir/threatbus/tree/master/apps/vast#bridge-features) for
+details. Or inspect the command help as follows.
 
 ```sh
-./vast-bridge --help
+./vast-bridge.py --help
+```
```

Comment:
I would not allocate an entire block just to show users how to print the help; that's commonly understood. I moved it as sentence fragment in the previous paragraph.